src/site/content/en/lighthouse-performance/resource-summary/index.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: Keep request counts low and transfer sizes small
+description: |
+  An overview of how high resource counts and large transfer sizes affect load performance, 
+  and strategies for reducing request counts and transfer sizes.
+updated: 2019-08-27
+web_lighthouse:
+  - resource-summary
+---
+
+The **Keep request counts low and transfer sizes small** audit tells you how many network requests 
+were made while your page loaded, as well as the total amount of data transferred.
+
+<figure class="w-figure">
+  <img class="w-screenshot w-screenshot--filled" src="resource-summary.jpg" 
+       alt="Keep request counts low and transfer sizes small">
+  <figcaption class="w-figcaption">
+    <b>Keep request counts low and transfer sizes small</b>
+  </figcaption>
+</figure>
+
+{% Aside %}
+  The **Requests** and **Transfer Size** values for the **Total** row is computed by adding the values
+  for the **Image**, **Script**, **Font**, **Stylesheet**, **Other**, **Document**, and **Media**
+  rows. The **Third-party** column does not factor into the **Total** row's values. Its purpose is
+  to make you aware of how many of the total requests and how much of the total transfer size came
+  from third-party domains. The third-party requests could be a combination of any of the other
+  resource types.
+{% endAside %}
+
+{% Aside %}
+  Like all of the **Diagnostics** audits, the **Keep request counts low and transfer sizes small**
+  audit does not directly affect your **Performance** score. However, reducing request counts or
+  transfer sizes may improve any or all of your **Performance** metrics, which does have a direct
+  effect on your **Performance** score.
+{% endAside %}
+
+## How high resource counts and large transfer sizes affect load performance
+
+The effect of high resource counts or large transfer sizes on load performance depends on what
+type of resource is being requested.
+
+### CSS and JavaScript
```

Comment:
Here's the pattern we discussed for this and the following subsections:
- Short para describing the problem
- 1-sentence para with link to post about how to optimize
- bulleted list of affected metrics